### ISSUE TITLE
how can i read qall

### ISSUE THREAD

**Issue by ts1cos** [syssi/esphome-pipsolar#46]
hi i would like to read qall and not qpigs where and how can i change that? have easun smr II and in qpigs not everything is sent eg discharge current

**Comment by SabolchGreba** [comment]
Do you solved this? I have a Victor NM-3200-24 which uses the same pattern as your inverter but I don't know how to edit the module.

**Comment by ts1cos** [comment]
No, I also tried solar2mqtt and failed with the same problem. now I have mpp-solar and it works.
